src/vue/backend/src/views/settings/General.vue

```diff
@@ -37,20 +39,23 @@ export default {
         return {
             error: false,
             loading: false,
-            success: false
+            success: false,
+            formDisabled: true
         }
     },
     methods: {
         async saveHandler () {
             // To Do: user facing error checking
             if (this.formChanges) {
+                this.formDisabled = true
```

Comment:
Formdisabled and loading are the same, so you can use form disabled in the api key component instead of updating both the loading and the formdisabled variables